src/js/lib/jquery.kazoosdk.js

```diff
@@ -558,6 +558,10 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			onebillIntegration: {
+				'get': { verb: 'GET', url: 'accounts/{accountId}/cdr_push_onebill' },
```

Comment:
What does this resource is expected to return, a single document or list of documents?

Reply:
This will return a single document